# Complete genome sequence of Acinetobacter baumannii MDR-CQ

Authors, Corresponding author*

Departments

  Acinetobacter baumannii is a pathogenic species of bacteria, identified as an aerobic Gram-negative bacterium, that is resistant to most antibiotics. In this study, the MDR-CQ strain was isolated at the Second Affiliated Hospital of Chongqing Medical University, China, and was found to be resistant to penicillin, cephalosporins, aminoglycosides, quinolones, and also imipenem. The genome sequences of Acinetobacter baumannii strain MDR-CQ was determined by using PacBio sequencing performed with the Pacbio RSII system to generate a complete assembly.

  Acinetobacter baumannii is a pathogenic species of bacteria, identified as an aerobic Gram-negative bacterium, that is resistant to most antibiotics (). As an important opportunistic pathogen, A. baumannii strains are an increasing cause of concern because of various reported nosocomial infections, which are often severe. In this paper, the MDR-CQ strain was isolated at the Second Affiliated Hospital of Chongqing Medical University, China, and was found to be resistant to penicillin, cephalosporins, aminoglycosides, quinolones, and also imipenem.

  The complete genome of A. baumannii strain MDR-CQ was determined on a PacBio RSII platform. To do this, a large library (20 to 23 kb) was constructed and sequenced, using one SMRT cell with a P6 polymerase and C4 chemistry comination (P6-C4) with a 180-min movie. The SMRT cells produced XXX post-filter polymerase reads with a mean read length of XXX bp. Subreads were assembled de novo using the PacBio RS hierarchical genome assembly process (HGAP) protocol version 3 in SMRT analysis version 2.0 (Pacific Bioscience) (). Functional annotation was performed with the NCBI Prokaryotic Genome Annotation Pipeline. 
  
  Currently, the complete genome, has a total of XXX bp, with an average G + C content of XXX%.The origin of replication (oriC) and nine putative DnaA boxes were identified by using Ori-Finder (). The regions with abnormal G + C contents in the genomic sequence were obtained by using the GC-Profile program () to identify the genomic resistance islands. Protein-coding genes were predicted by using three ab initio gene-finding programs: Glimmer V3.02 (), GeneMark (), and ZCURVE v2.0 (). Seventeen rRNA genes (five 5S rRNAs, six 16S rRNAs, and six 23S rRNAs) of the draft assembly were identified by using RNAmmer (). Seventy-four tRNA genes for all 20 amino acids were predicted by using tRNAscan-SE (). Automatic functional annotation results and subsystems represented in the genome were obtained by using the Rapid Annotation using Subsystem Technology (RAST) server. Consequently, 78 subsystem features predicted by the RAST server were relevant to resistance to antibiotics and toxic compounds, which will be helpful for revealing the mechanisms of Acinetobacter baumannii strain MDR-CQ's high resistance to almost all antibiotics.

  **Nucleotide sequence accession numbers. ** The information reported here from the Whole Genome Shotgun project has been deposited with DDBJ/EMBL/GenBan under the accession number XXX. The version discribed in this paper is the first version and has been assigned accession number XXX.
  
  We thank XXX.
  
  The present work was partially supported by the XXX (grants XXX, XXX) and XXX (grant XXX).
  
REFERENCES
